### PR TITLE
Fix #248: Create standard API Docs for Admin Part

### LINKED ISSUE #248: Create standard API Docs for Admin Part

### PR DESCRIPTION
**Pull request by jandusil**

### CODE REVIEW

**Comment by jandusil** [comment]
Converting to draft, found some issues

**Review by banterCZ — COMMENTED** [review]
I believe that the same change as in `_Sidebar.md_ should be made in `Readme.md` as well.

File: docs/Readme.md [hunk 1]
```diff
@@ -1,17 +1,7 @@
...
-- [Deployment](./Deployment.md)
-- [Database Structure](./Database-Structure.md)
-- [Migration Instructions](./Migration-Instructions.md)
-- [Configuration](./Configuration.md)
-- [Public RESTful API](./Public-REST-API.md)
-<!-- end -->
-
-The Mobile Utility Server by [Wultra](https://wultra.com) is a utility server with various features required by typical mobile apps.
+The Mobile Utility Server by [Wultra](https://wultra.com) is a utility server with various features required by typical
+mobile apps.
```
Comment (zcgandcomp) [inline docs/Readme.md:4]: Why is the new line here?

File: docs/Readme.md [hunk 2]
```diff
@@ -20,7 +10,8 @@ Currently, the feature set is relatively narrow:
 
 ## License
 
-Mobile Utility Server is licensed using GNU AGPLv3 license. Please consult us at [hello@wultra.com](mailto:hello@wultra.com) for the software use.
+Mobile Utility Server is licensed using GNU AGPLv3 license. Please consult us
+at [hello@wultra.com](mailto:hello@wultra.com) for the software use.
```
Comment (zcgandcomp) [inline docs/Readme.md:14]: Why is the new line here?

**Review by zcgandcomp — CHANGES_REQUESTED** [review]

**Review by zcgandcomp — APPROVED** [review]
Looks OK

**Pull request merged**